inst/extdata/README.md

```diff
@@ -104,6 +104,38 @@ Columns:
 | unit | units used for numeric data. |
 
 
+## [validation_summary.tsv](./validation_summary.tsv)
```

Comment:
After reading https://contributions.bioconductor.org/docs.html#doc-inst-script, there should be documentation about how to recreate the data.